### ISSUE TITLE
Use of cosmetic fragrances

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10149]
**Preferred term label:**
Use of cosmetic fragrances

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C0577618
https://uts.nlm.nih.gov/uts/umls/concept/C4761084

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Riehl melanosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
according to https://www.ncbi.nlm.nih.gov/books/NBK557437/
it is not just fragrances, but also dyes.
Possibly Exposure to cosmetic allergens or Use of cosmetic products
?

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000636
- label: Exposure to cosmetic fragrances
- parents: [Exposure history (HP:4000110)](https://hpo.jax.org/app/browse/term/HP:4000110)
- pmid: 
- def: A history of exposure to fragrances found in cosmetic products.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10149
- synonyms:
